docs/pages/includes/database-access/db-configure-start.mdx

```diff
@@ -0,0 +1,127 @@
+{{ dbName="test" }}
```

Comment:
Should we have a default value for the other variables as well?

Reply:
We've had a mix of guides using variables for the proxy so the user can specify.  Not sure how far we want to go with that.  We could certainly set the proxy and other variables here.

Reply:
After a deeper look, I don't think we need to set a default value for the other vars. But I do wonder if we should explicitly set `dbName` everywhere we use this file? On 4 pages we call this partial without specifying it, resulting in a config that creates a "test" name configuration.

Reply:
I approved the PR, but would still like to see if we can resolve this concern.

Reply:
addressed